cvat-core/src/annotations.ts

```diff
@@ -161,20 +162,23 @@ export function importDataset(
     file: File | string,
     options: {
         convMaskToPoly?: boolean,
-        updateStatusCallback?: (s: string, n: number) => void,
+        requestStatusCallback?: (request: Request) => void,
+        uploadStatusCallback?:(s: string, n: number) => void,
     } = {},
-): Promise<void> {
-    const updateStatusCallback = options.updateStatusCallback || (() => {});
+): Promise<string> {
+    const uploadStatusCallback = options.uploadStatusCallback || (() => {});
+    const requestStatusCallback = options.requestStatusCallback || (() => {});
     const convMaskToPoly = 'convMaskToPoly' in options ? options.convMaskToPoly : true;
     const adjustedOptions = {
-        updateStatusCallback,
+        requestStatusCallback,
```

Comment:
We need a check for this new argument like for `uploadStatusCallback`
Message should be more specific if we have many callbacks now:

``throw new ArgumentError('Callback must be a function');`` ->  ``throw new ArgumentError('Request status callback must be a function');``